content/W2016/W16-event-blogging.md

```diff
@@ -0,0 +1,32 @@
+Title: Big CSters Blogging Workshop
+Date: 2016-02-23 19:00
+Category: Events
+Tags: social, big csters, blogging
+Slug: blogging-workshop
+Author: Julia Nguyen
+Summary: Join us for an interactive workshop on blogging!
+
+Interested in technical writing, blogging, or crafting your online identity? 
+Join us for an interactive workshop on blogging! This workshop will help you hone your writing 
+- from generating ideas to finding the right publishing platform! 
```

Comment:
Now that this line has been split, starting it with a `-` renders this as a bullet point (see https://github.com/julianguyen/wics-website/blob/master/content/W2016/W16-event-blogging.md)